### PR TITLE
docs: added examples for awscc_qbusiness_web_experience

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

**Review by BondAnthony — APPROVED** [review]
Thank you!

File: examples/resources/awscc_qbusiness_web_experience/qbusiness_web_experience.tf [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            "aws:SourceAccount" = data.aws_caller_identity.current.account_id
+          }
+          ArnEquals = {
+            "aws:SourceArn" = "arn:${data.aws_partition.current.id}:qbusiness:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}}:application/*"
+          }
+        }
+      }
+    ]
+  })
+
```
Comment (wellsiau-aws) [inline examples/resources/awscc_qbusiness_web_experience/qbusiness_web_experience.tf:42]:
```suggestion
  policies = [{
    policy_name = "qbusiness_policy"
    policy_document = jsonencode({
      Version = "2012-10-17"
      Statement = [
        {
          Sid = "QBusinessConversationPermission"
          Effect = "Allow"
          Action = [
            "qbusiness:Chat",
            "qbusiness:ChatSync",
            "qbusiness:ListMessages",
            "qbusiness:ListConversations",
            "qbusiness:DeleteConversation",
            "qbusiness:PutFeedback",
            "qbusiness:GetWebExperience",
            "qbusiness:GetApplication",
            "qbusiness:ListPlugins",
            "qbusiness:GetChatControlsConfiguration"
          ]
          Resource = awscc_qbusiness_application.example.application_arn
        }
      ]
    })
```

missing policies for the IAM role

File: examples/resources/awscc_qbusiness_web_experience/qbusiness_web_experience.tf [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Action = [
+          "sts:AssumeRole",
+          "sts:SetContext"
+        ]
+        Condition = {
+          StringEquals = {
+            "aws:SourceAccount" = data.aws_caller_identity.current.account_id
+          }
+          ArnEquals = {
+            "aws:SourceArn" = "arn:${data.aws_partition.current.id}:qbusiness:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}}:application/*"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_qbusiness_web_experience/qbusiness_web_experience.tf:36]:
```suggestion
            "aws:SourceArn" = awscc_qbusiness_application.example.application_arn
```

I noticed that we can scope it down further to the exact ARN

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
please check the suggested changes

**Review by wellsiau-aws — APPROVED** [review]

**Comment by quixoticmonk** [comment]
> please check the suggested changes
Updated the suggested changes.

**Comment by marcosentino** [comment]
@quixoticmonk Thank you for the contribution!

**Pull request merged**